### PR TITLE
feat: stride spec add

### PR DESCRIPTION
**Pull request by BabyScope**
testscript : ./scripts/test_spec_full.sh cookbook/specs/spec_add_stride.json tendermintrpc https://stride.rpc.kjnodes.com/ rest https://stride.api.kjnodes.com grpc stride.grpc.kjnodes.com:443 tendermintrpc https://stride-testnet-rpc.polkachu.com/ rest https://stride-testnet-api.polkachu.com/ grpc stride-testnet-grpc.polkachu.com:12290

[Portal Stride.txt](https://github.com/lavanet/lava/files/15287658/Portal.Stride.txt)
[Provider Stride.txt](https://github.com/lavanet/lava/files/15287659/Provider.Stride.txt)

### CODE REVIEW

File: cookbook/specs/stride.json [hunk 1]
```diff
@@ -0,0 +1,1848 @@
...
+    "proposal": {
+      "title": "Add Specs: Stride",
+      "description": "Adding new specification support for relaying Stride data on Lava",
+      "specs": [
+        {
+          "index": "STRIDE",
+          "name": "stride mainnet",
+          "enabled": true,
+          "imports": [
+            "COSMOSSDK"
```
Comment (Yaroms) [inline cookbook/specs/stride.json:11]: does is not support cosmwasm?
COSMOSSDKFULL -> with cosmwasm
COSMOSSDK -> without cosmwasm

File: cookbook/specs/stride.json [hunk 2]
```diff
@@ -0,0 +1,1848 @@
...
+                    {
+                      "expected_value": "stride-1"
+                    }
+                  ]
+                },
+                {
+                  "name": "pruning",
+                  "values": [
+                    {
+                      "latest_distance": 14900
```
Comment (Yaroms) [inline cookbook/specs/stride.json:1694]: we are missing archive verification here

File: cookbook/specs/stride.json (hunk 1 above)
Comment (BabyScope) [reply cookbook/specs/stride.json:11]: Nope Stride doesn't support wasm

File: cookbook/specs/stride.json (hunk 2 above)
Comment (BabyScope) [reply cookbook/specs/stride.json:1698]: Resolved does the archive verification need to be applied on all i did it only on tendermint?

Comment (Yaroms) [reply cookbook/specs/stride.json:1698]: yes, should be applied on all of the interfaces

**Review by Yaroms — APPROVED** [review]

**Pull request merged**